components/next_steps_view/steps/enable_notifications_step/enable_notifications_step.test.tsx

```diff
@@ -19,7 +21,11 @@ describe('components/next_steps_view/steps/setup_preferences_step', () => {
     };
 
     test('should match snapshot', () => {
-        const wrapper = shallow(<EnableNotificationsStep {...props}/>);
+        const store = testConfigureStore();
+
+        const wrapper = mountWithIntl(<Provider store={store}>
```

Comment:
I had to replace `shallow` with `mount` here since `useDispatch` does not seem to work happily with Enzyme 16 shallow renderer. As a result, the snapshot has got bigger. Let me know if this may be a concern.